src/test/scala/cromwell/binding/ScatterSpec.scala

```diff
@@ -0,0 +1,70 @@
+package cromwell.binding
+
+import cromwell.parser.BackendType
+import cromwell.util.SampleWdl
+import org.scalatest.{FlatSpec, Matchers}
+
+class ScatterSpec extends FlatSpec with Matchers {
+  val namespace = NamespaceWithWorkflow.load(SampleWdl.ScatterWdl.wdlSource(), BackendType.LOCAL)
+
+  it should "Have four 'children' objects" in {
+    namespace.workflow.children.size shouldEqual 4
+  }
+
+  it should "Have two 'direct Call descendents' objects" in {
+    Scope.collectCalls(namespace.workflow.children).size shouldEqual 2
+  }
+
+  it should "Have two 'direct Scatter descendents' objects" in {
+    Scope.collectScatters(namespace.workflow.children).size shouldEqual 2
+  }
+
+  it should "Have eight 'Call' objects" in {
+    namespace.workflow.calls.size shouldEqual 8
+  }
+
+  it should "Have four 'Scatter' objects" in {
+    namespace.workflow.scatters.size shouldEqual 4
+  }
+
+  it should "Have 'Scatter' objects indexed properly" in {
+    namespace.workflow.scatters(0).index shouldEqual 0
+    namespace.workflow.scatters(1).index shouldEqual 3
+    namespace.workflow.scatters(2).index shouldEqual 1
+    namespace.workflow.scatters(3).index shouldEqual 2
```

Comment:
Right this looks a lot better than my version, I'm also probably not familiar enough with all scalaTest features yet.